examples/srd_v1_3x2.yaml

```diff
@@ -4,37 +4,38 @@ generate:
     # will once be part of TJPCov interface
     fsky : &fsky 0.35
     add_noise: &add_noise False # add a realization of noise?
-      
+    kmax: &kmax 0.3 # kmax in Mpc^-1 for 2pt analyses
+
   parameters: &parameters
     Omega_k: 0.0
     Omega_b: 0.0492
     Omega_c: 0.26639999999999997  # = 0.3156 - 0.0492
     w0: -1.0
     wa: 0.0
     h: 0.6727
-    sigma8: 0.83 
+    sigma8: 0.83
     n_s: 0.9645
-  
+    transfer_function: 'eisenstein_hu'
```

Comment:
is there a reason to use eisenstein_hu here rather than camb or class? just curious

Reply:
It was to avoid problems with CAMB for certain combinations of w0-wa